script/release/version-bumper.js

```diff
@@ -61,6 +61,11 @@ async function main () {
     updateWinRC(components)
   ]);
 
+  // if bumping stable, update supported versions
+  if (opts.bump === 'stable') {
```

Comment:
Stable !== "promote a beta to stable" all non-beta releases are considered "stable" so this check would mean a release of `10.4.0` (even if 11, 12 and 13) exist would update the supported versions document.  That _might_ be what we want given this document is kept in each version branch.

cc @erickzhao Which version of this document is deployed to the website, is it the version on the latest stable branch, or the version on master?

Reply:
Ahhh, okay - I don't think that's what we'd want to do here, then, since the supported versions should always support the latest three versions. I'll retool this check.